Formula/confluent-hub-client.rb

```diff
@@ -0,0 +1,13 @@
+class ConfluentHubClient < Formula
+  desc "Confluent Hub Client"
+  homepage "https://confluent.io"
+  url "https://s3-us-west-2.amazonaws.com/confluent-hub-client/confluent-hub-client-5.1.0-beta180619025141-package.tar.gz"
```

Comment:
How is the version string get updated with every release? From what I can tell there's not templating in the code. Are we planning to use github releases somehow?

Reply:
It's a manual process now. I've evaluated GitHub releases as a potential place to host our releases, but it wouldn't work for COS users since this repo is private. So I ended up using a public S3 bucket to host the release and manually updating the Formula.